includes/Entity/class-item.php

```diff
@@ -282,15 +282,29 @@ public function get_author( $property = null ) {
 	 * @return string
 	 */
 	public function get_content() {
-		if ( $this->content ) {
+		$text_len = $this->str_length( $this->content );
+
+		if ( ( 0 === $text_len ) ) {
+			if ( ! empty( $this->summary ) ) {
```

Comment:
It think `if ( $this->summary ) {` should do the exact same and maybe a bit easier to read?!?

Reply:
Okay. I'll put it back to just $this->summary, but what if it isn't a string? Probably should check that too.

Reply:
is that really possible?